src/test/algorithms/searching/ternary_search.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+require('chai').should();
+
+var ternarySearch = require('../../..').Search.ternarySearch,
+    eps = 1e-6;
+
+var fn1 = function (x) {
+  return -Math.pow(x - 2, 2) + 4;
+};
+
+var fn2 = function (x) {
+  return -2 * Math.cos(x);
+};
+
```

Comment:
Add a function here:
```javascript
var closeEnough = function (a, b, precision) {
  return Math.abs(a - b) < precision;
};
```

And use it instead of requiring `chai`:

```javascript
assert(closeEnough(ternarySearch(...), expected, eps)
```